staging/jobs/pingcap/tiflow/latest/ghpr_verify.groovy

```diff
@@ -17,7 +17,7 @@ pipelineJob('pingcap/tiflow/ghpr_verify') {
                     cron('H/5 * * * *')
                     gitHubAuthId('') // using the default only one.
 
-                    triggerPhrase('.*/(run-(debug-verify).*)')
+                    triggerPhrase('.*/((run-debug-verify|verify).*)')
```

Comment:
when prod job ready, we should disable the staging job, rest same suggestion.

Reply:
yes, i will diable those staging job after prod is ready to work